reactor-core/src/test/java/reactor/core/publisher/NextProcessorTest.java

```diff
@@ -327,7 +329,7 @@ public void mapFulfill() {
 		mp.onNext(1);
 
 		NextProcessor<Integer> mp2 = mp.map(s -> s * 2)
-									   .cache()
+		                               .cache()
```

Comment:
nitpick: I think IntelliJ's formatter is getting confused by the mix of tabs and spaces in this file. we should generally favor tabs for most of the colums (and let it adjust the alignment of dots with a few spaces if there is a shift of less than 4 characters)